app/services/contact_message_service.rb

```diff
@@ -1,8 +1,10 @@
 class ContactMessageService
+  def self.deliver(email:, message:)
+    data = { from: { type: "user", email: email }, body: message }
+    headers = {'Authorization' => "Bearer #{ENV['INTERCOM_ACCESS_TOKEN']}",
+               content_type: :json,
+               accept: :json}
 
-  def self.create(contact_message:, actor:)
-    contact_message.user = actor
-    ContactMessageMailer.delay(priority: 2).contact_message_email contact_message if contact_message.save
+    RestClient.post("https://api.intercom.io/messages", data.to_json, headers)
```

Comment:
I'll hold off on my desire to transform this into `Clients::Intercom`, since this is clearly a bandaid until we can get unhooked from Intercom.

I'll note though that the express purpose of a `Clients::Base` subclass is 'a model which sends http requests to a specific api', where we define a base api (`api.intercom.io`), a path (`messages`), some data / headers to send, and some behaviour to do when it responds.